localenv/cloud-nine-wallet/docker-compose.yml

```diff
@@ -49,6 +49,7 @@ services:
       AUTH_SERVER_INTROSPECTION_URL: http://cloud-nine-wallet-auth:3007
       ILP_ADDRESS: test.cloud-nine-wallet
       STREAM_SECRET: BjPXtnd00G2mRQwP/8ZpwyZASOch5sUXT5o0iR5b5wU=
+      SLIPPAGE: 0
```

Comment:
controversial, but maybe this makes more sense in a single peer environment? this basically takes the network fees down to 0 on the quote (since we assume no FX change variation). Ccould be more straightforward/easier to understand in the localenv setup especially. Would be curious on other's opinions

Reply:
I'm thinking of sticking with the default value of `0.01` for the slippage. This way we can show possible fluctuations and that they can happen.

Reply:
Yep, reverted